src/System.Management.Automation/engine/InternalCommands.cs

```diff
@@ -585,6 +585,10 @@ private void EndParallelParameterSet()
             _taskCollection.Complete();
```

Comment:
@PaulHigin, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '10.79.1.177', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 257
257:             $result[0].TargetAddress | Should -BeExactly $realAddress
```